packages/field-base/src/input-control-mixin.js

```diff
@@ -166,6 +166,10 @@ export const InputControlMixin = (superclass) =>
       input.addEventListener('beforeinput', this._boundOnBeforeInput);
     }
 
+    __targetIsInput(event) {
+      return event.target === this.inputElement;
```

Comment:
IMO we don't need this method as it's only used in one place, I would rather inline its code to the `if` statement.